Dockerfile

```diff
@@ -1,38 +1,47 @@
 FROM docker.io/golang:1.24-bookworm as builder
 
 ARG VER=devel
-ARG BUILDTAGS=""
-ARG EXTERNAL="0"
+ARG BUILDTAGS
```

Comment:
can `docker build .` work after default value removed

Reply:
Yes? `""` is empty, it should be the same as when you use the ARG, it's not required to be set. Up to you what you prefer though 👍